test/performance/statsd/parameters.yml

```diff
@@ -0,0 +1,71 @@
+receivers: "statsd"
```

Comment:
The validator will need a generator configuration (`parameters.yml`) to know about the test case information (e.g [what metrics needs to validated or submitted to database](https://github.com/aws/amazon-cloudwatch-agent-test/pull/155/files#diff-61c2194d8a03b011eb19636e25cac8c60ad20430d5d095ad5016868205da296fR18-R71), what is the [amount of run time that agent needs to run](https://github.com/aws/amazon-cloudwatch-agent-test/pull/155/files#diff-61c2194d8a03b011eb19636e25cac8c60ad20430d5d095ad5016868205da296fR10))